src/test/java/de/tum/in/www1/artemis/FileUploadExerciseIntegrationTest.java

```diff
@@ -62,11 +67,19 @@ public void createFileUploadExercise() throws Exception {
         String filePattern = "png, pdf, jPg      , r, DOCX";
         FileUploadExercise fileUploadExercise = database.createFileUploadExercisesWithCourse().get(0);
         fileUploadExercise.setFilePattern(filePattern);
+        gradingCriteria = database.addGradingInstructionsToExercise(fileUploadExercise);
         FileUploadExercise receivedFileUploadExercise = request.postWithResponseBody("/api/file-upload-exercises", fileUploadExercise, FileUploadExercise.class);
 
         assertThat(receivedFileUploadExercise).isNotNull();
         assertThat(receivedFileUploadExercise.getId()).isNotNull();
         assertThat(receivedFileUploadExercise.getFilePattern()).isEqualTo(filePattern.toLowerCase().replaceAll("\\s+", ""));
+
+        assertThat(receivedFileUploadExercise.getGradingCriteria().get(0).getTitle()).isEqualTo(null);
```

Comment:
Just a nitpick, but I saw that you have this Assertion (and the following ones) in the method `testGetModelingExercise_asTA` as well. You could actually move it into your method `addGradingInstructionsToExercise`, so you don't have to add the Assertion every time you call `addGradingInstructionsToExercise`